src/main/java/org/folio/circulation/services/LostItemFeeRefundService.java

```diff
@@ -44,7 +44,15 @@ public LostItemFeeRefundService(Clients clients) {
     CheckInContext checkInContext) {
 
     return refundLostItemFees(forCheckIn(checkInContext))
-      .thenApply(r -> r.map(context -> checkInContext.withLoan(context.getLoan())));
+      .thenApply(r -> r.map(context -> {
+        // check-in of item without an open loan (the only possible case is Lost and paid items)
+        // for such check-in we should not set loan in response.
```

Comment:
Is this comment suggesting that the only circumstances where an item is checked in is when it is lost and paid?

It is my understanding that an item can be checked in at any point, and mod-circulation should tolerate it. In some cases, I think it is required for processes to complete, e.g. in transit for hold shelf requests.

Reply:
Yes, I should adjust the comment. All item statuses is allowed and should be correctly handled.